README.md

```diff
@@ -11,7 +11,7 @@ Please follow the [Python Extension ReadMe](https://github.com/microsoft/vscode-
 
 ## Working with other Languages
 
-The Jupyter Extension supports other languages such as Julia and R (as well as Python) in VS Code Insiders with our new Native Notebooks Experience!
+The Jupyter Extension supports other languages in addition to Python such as Julia, R, and C# in VS Code Insiders with our latest Native VS Code Notebooks Experience!
```

Comment:
Github won't let me place a comment there, but on line 8 there are two spaces between "partnership" and "of the" that could be fixed while we are fixing this.

Reply:
This is just an Ian suggestion but I'd rather be more sparing with "!". Doesn't seem super appropriate here.